modules/rosa-adding-taints.adoc

```diff
@@ -14,7 +14,7 @@ You can add taints for compute (also known as worker) nodes in a machine pool to
 
 ifdef::openshift-rosa[]
 * You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
-* You logged in to your Red Hat account by using the `rosa` CLI.
+* You logged in to your Red Hat account by using the {product-title} (ROSA) CLI (`rosa`).
```

Comment:
Same fixes suggested for https://github.com/openshift/openshift-docs/pull/61151/files#r1233482587